subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -638,6 +643,10 @@ static void delayed_action_timeout(struct k_work *work)
 		.time = srv->fade.duration
 	};
 
+	if (!is_enabled(srv)) {
```

Comment:
It doesn't seem related to delayable work API changes :)

Reply:
It is. We can not assume cancel() succeeds, and need to exit early in case it failed